### ISSUE TITLE
Deep Learning 5

### ISSUE THREAD

**Issue by p4sscode** [p4sscode/hdc#10]
- [x] 5.1 Learning Algorithms => 99
- [ ] 5.2 Capacity, Overﬁtting and Underﬁtting => 110
- [ ] 5.3 Hyperparameters and Validation Sets => 120
- [ ] 5.4 Estimators, Bias and Variance => 122
- [ ] 5.5 Maximum Likelihood Estimation => 131
- [ ] 5.6 Bayesian Statistics => 135
- [ ] 5.7 Supervised Learning Algorithms => 140
- [ ] 5.8 Unsupervised Learning Algorithms => 146
- [ ] 5.9 Stochastic Gradient Descent => 151
- [ ] 5.10 Building a Machine Learning Algorithm => 153
- [ ] 5.11 Challenges Motivating Deep Learning => 155
